patches/content-browser-loader-browser_initiated_resource_request.cc.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/content/browser/loader/browser_initiated_resource_request.cc b/content/browser/loader/browser_initiated_resource_request.cc
+index d9cbc455a65aaea690b8c35e64164dc282740110..0fb711fa9eaff907c80d0ef1af714a74af972054 100644
+--- a/content/browser/loader/browser_initiated_resource_request.cc
++++ b/content/browser/loader/browser_initiated_resource_request.cc
+@@ -5,7 +5,10 @@
+ #include "content/browser/loader/browser_initiated_resource_request.h"
+ 
+ #include "base/command_line.h"
++#include "base/feature_list.h"
+ #include "base/metrics/field_trial_params.h"
++#include "brave/common/brave_features.h"
++#include "brave/common/network_constants.h"
+ #include "content/common/content_constants_internal.h"
+ #include "content/public/browser/browser_context.h"
+ #include "content/public/browser/browser_thread.h"
+@@ -37,6 +40,9 @@ void UpdateAdditionalHeadersForBrowserInitiatedRequest(
+     }
+     headers->SetHeaderIfMissing(kDoNotTrackHeader, "1");
+   }
++  if (base::FeatureList::IsEnabled(features::kGlobalPrivacyControl)) {
```

Comment:
this is not the right place to do this. Please see the network delegate helper classes